File: shoebill_plugins/JobPlugin/src/main/java/lt/ltrp/FactionInviteOffer.java
package lt.ltrp;

import lt.ltrp.data.PlayerOffer;
import lt.ltrp.object.LtrpPlayer;
import net.gtaun.util.event.EventManager;

/**
 * @author Bebras
 *         2016.03.04.
 */
public class FactionInviteOffer extends PlayerOffer {


    public FactionInviteOffer(LtrpPlayer player, LtrpPlayer offeredBy, EventManager eventManager) {
        super(player, offeredBy, eventManager, 120, FactionInviteOffer.class);
    }
}

File: shoebill_libs/LtrpApi/src/main/java/lt/ltrp/player/job/data/FactionInviteOffer.kt
package lt.ltrp.player.job.data

import lt.ltrp.data.PlayerOffer
import lt.ltrp.`object`.LtrpPlayer
import lt.ltrp.job.`object`.Faction
import net.gtaun.util.event.EventManager

/**
 * @author Bebras
 *         2016.03.04.
 */
class FactionInviteOffer(player: LtrpPlayer, offeredBy: LtrpPlayer, val faction: Faction, eventManager: EventManager):
        PlayerOffer(player, offeredBy, eventManager, 120, FactionInviteOffer::class.java) {

}
